sql/firefox_desktop_exact_mau28_by_dimensions_v1.sql

```diff
@@ -1,18 +1,15 @@
-WITH
-  inactive_days AS (
-    SELECT
-      *,
-      DATE_DIFF(submission_date, last_seen_date, DAY) AS _inactive_days
-    FROM
-      clients_last_seen_v1
-  )
-
 SELECT
-  submission_date,
-  CURRENT_DATETIME() AS generated_time,
-  COUNTIF(_inactive_days < 28) AS mau,
-  COUNTIF(_inactive_days < 7) AS wau,
-  COUNTIF(_inactive_days < 1) AS dau,
+  @submission_date AS submission_date,
+  COUNTIF(days_since_seen < 28) AS mau,
+  COUNTIF(days_since_seen < 7) AS wau,
+  COUNTIF(days_since_seen < 1) AS dau,
+  -- Active MAU counts all Active Users on any day in the last 28 days not just
+  -- the most recent day making COUNTIF(_days_since_seen < 28 AND active_user)
+  -- incorrect. Instead we track days_since_active_user and use that.
+  -- https://docs.telemetry.mozilla.org/cookbooks/active_dau.html
+  COUNTIF(days_since_active_user < 28) AS active_mau,
+  COUNTIF(days_since_active_user < 7) AS active_wau,
+  COUNTIF(days_since_active_user < 1) AS active_dau,
```

Comment:
Assuming we change "active_user" terminology, these might become "dau_5_uri_active", etc.